constellation_orderboard/views.py

```diff
@@ -10,9 +11,14 @@
 from django.urls import reverse
 from django.contrib.auth.decorators import (
```

Comment:
This is now a one line import, the grouping control characters should be removed.